modules/social_features/social_search/social_search.install

```diff
@@ -227,3 +227,17 @@ function social_search_update_10002() {
     \Drupal::logger('social_search')->info($e->getMessage());
   }
 }
+
+/**
+ * Add translation support for search views.
+ */
+function social_search_update_10003() {
```

Comment:
@nechai As we agreed better to start hook update with the target OS version, so it will be `10201`

Reply:
@ribel Resolved (changed destination branch to 10.3.x)